packages/commands/src/zetachain/pools/liquidity/add.ts

```
@@ -0,0 +1,286 @@
+     * 3. Parse token addresses from CLI
+     */
+    if (validatedOptions.tokens.length !== 2) {
+      throw new Error("Exactly 2 token addresses must be provided");
+    }
+
+    const [inputTokenA, inputTokenB] = validatedOptions.tokens.map((addr) =>
+      ethers.getAddress(addr)
+    );
+
+    /**
+     * 4. Locate (or verify) the pool — order agnostic
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)